src/hdmf_zarr/backend.py

```diff
@@ -531,7 +541,13 @@ def __resolve_ref(self, zarr_ref):
             try:
                 target_zarr_obj = target_zarr_obj[object_path]
             except Exception:
-                raise ValueError("Found bad link to object %s in file %s" % (object_path, source_file))
+                try:
+                    import pathlib
```

Comment:
@oruebel This is the update to `resolve_ref` I mentioned below. This checks that the relative_path works to retrieve the target. This would change/be updated as we bring zarr up to speed with the "queue" HDMF uses so that we don't need this check.